src/store/ChartStore.ts

```diff
@@ -54,7 +48,7 @@ type TSpan = {
     maintainPeriodicity?: boolean;
     padding?: number;
     forceLoad?: boolean;
-    chart?: typeof CIQ.ChartEngine.Chart;
+    chart?: any;
```

Comment:
@balakrishna-deriv Are the `chart` variable and `TRange` and `TSpan` types still used anywhere? it seems like they are no longer used in this file.

Reply:
@maryia-deriv I've removed those. Thanks for checking 🙏